src/mono/wasm/runtime/startup.ts

```diff
@@ -707,10 +717,11 @@ export function mono_wasm_load_data_archive(data: Uint8Array, prefix: string): b
  */
 export async function mono_wasm_load_config(configFilePath: string): Promise<void> {
     const module = Module;
+    const resolvedConfigFilePath = is_path_absolute(configFilePath) ? configFilePath : locateFile(configFilePath);
```

Comment:
I didn't do this detection in my PR. I guess we always only supported relative paths for all resources so far ? Is there strong scenario for absolute path ?

Reply:
For example, when used on the web, I thought there was a use case for distributing resources such as assemblies at different origins.
However, since there are more use cases where the AppBundle directory including dotnet.wasm is distributed as a single entity, it seems unnecessary to determine the absolute path.

Reply:
okay, this is my take on it 
https://github.com/dotnet/runtime/blob/75192bbc6f06f6471bc9d3c3e4ffbaad19020394/src/mono/wasm/runtime/imports.ts#L182